webdriver/client.py

```diff
@@ -385,12 +377,13 @@ def send_command(self, method, uri, body=None):
 
         url = urlparse.urljoin("session/%s/" % self.session_id, uri)
         response = self.send_raw_command(method, url, body)
+        value = response.body["value"]
 
-        rv = response.body["value"]
-        if not rv:
-            rv = None
+        if response.status != 200:
```

Comment:
The intention with `send_raw_command` is to return a `transport.Response` object that can be asserted using `wd_asserts`, if I understand @jugglinmike’s thinking correctly. It would defeat the purpose of the assertions if `send_raw_command` would raise.

Maybe we can change `send_command` to check if `self.session_id` is defined, before constructing `url`, so that starting and ending the session can use `send_command` instead of `send_raw_command`?